src/otx/core/model/entity/classification.py

```diff
@@ -63,7 +64,9 @@ def __init__(self, config: DictConfig) -> None:
         super().__init__()
 
     def _create_model(self) -> nn.Module:
-        return _create_mmpretrain_model(self.config, self.load_from)
+        model, classification_layers = _create_mmpretrain_model(self.config, self.load_from)
+        self.classification_layers = classification_layers
```

Comment:
Why don't we create them line by line?
```suggestion
        model = _create_mmpretrain_model(self.config, self.load_from)
        self.classification_layers = get_classification_layers(self.config)
```

Reply:
get_classification_layers funciton require model registry to build model with different num classes